win32ss/user/ntuser/defwnd.c

```diff
@@ -697,6 +697,29 @@ IntDefWindowProc(
          UserDerefObjectCo(Wnd->spwndParent);
          break;
 
+      case WM_POPUPSYSTEMMENU:
+      {
+         /* This is an undocumented message used by the windows taskbar to
+            display the system menu of windows that belong to other processes. */
+         USER_REFERENCE_ENTRY MenuRef;
+         PMENU pMenu;
+
+         ERR("WM_POPUPSYSTEMMENU\n");
```

Comment:
Add a comment about why/until_when this "error" debug log is wanted?

Reply:
Comment added.